Views/Home/Index.cshtml

```diff
@@ -1,16 +1,53 @@
+@using VoteMyst.Database
+
+@model FeaturedEvents
 @{
     ViewData["Title"] = "Index";
     Layout = "_Layout";
 }
 
-<div class="welcome-container">
```

Comment:
No more welcome message?

Reply:
I thought we'd do it like itch ...

> id say do it like itch, show the most recent upcoming and ongoing events.

.. which doesn't show a welcome message. Wouldn't that just take the attention away from the events?